indico/modules/events/api.py

```diff
@@ -617,7 +617,7 @@ def _build_event_api_data(self, event):
             'creationDate': self._serialize_date(event.created_dt),
             'creator': self._serialize_person(event.creator, person_type='Avatar', can_manage=can_manage),
             'hasAnyProtection': event.effective_protection_mode != ProtectionMode.public,
-            'roomMapURL': event.room.map_url if event.room else None,
+            'roomMapURL': event.room.map_url if event.room else event.map_url,
```

Comment:
```suggestion
            'roomMapURL': event.map_url,
```

This should be enough considering the implementation of `map_url`:

```python
    @property
    def map_url(self):
        if self.own_map_url:
            return self.own_map_url
        elif not self.room:
            return ''
        return self.room.map_url or ''
```

(I think it makes sense that the custpm map URL is always prioritized over the default one)